docker-compose.yml

```
@@ -52,6 +52,32 @@ services:
 
+  frontend-prod:
+    build:
+      context: frontend
+      dockerfile: Dockerfile
+      args:
+        - DEV=false
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./frontend:/app
+      - /app/node_modules # to avoid overwriting node_modules
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)